runner/Tests/maya/render-arnold-linux-container/job.parameters.2020.json

```diff
@@ -1,15 +1,15 @@
 {
    "jobName":{
-      "value":"container-centos-maya2018-arnold"
+      "value":"container-centos-maya2020-arnold"
    },
    "poolId":{
-      "value":"container-centos-maya2018-arnold"
+      "value":"container-centos-maya2020-arnold"
    },
    "containerImage":{
       "value": "azurebatchrendering/centos-maya_2018u4_arnold2023:latest"
```

Comment:
this will fail - need to remove support for it unless we are planning on pushing a new containerImage